flask/tests/test_madmin.py

```diff
@@ -0,0 +1,136 @@
+import pytest
+from app.madmin import MinioAdmin, readonly_buckets_policy, readwrite_buckets_policy
```

Comment:
I can't seem to run these tests because `app` module is not found. How are you running this?

Reply:
You have to do `python -m pytest` or `./utils/run.sh test`, assuming you installed dev dependencies in a virtualenv. For some reason, running pytest as a module works for module resolution but not `pytest`. If you're just using Docker then I'm working on getting tests up inside that on a separate branch for #122. Make sure that when you run these tests, you haven't added any custom users, groups, or policies to the MinIO instance.

Reply:
"For some reason, running pytest as a module works for module resolution but not `pytest`"
What on earth?!?
Well I ran it with `python -m pytest` and the tests passed so looks good